static/styles/settings.scss

```diff
@@ -481,15 +481,25 @@ input[type=checkbox] {
     color: hsl(0, 0%, 47%);
 }
 
+#settings_page .icon-button.primary {
+    background-color: hsl(156, 30%, 50%);
+    color: hsl(0, 0%, 100%);
```

Comment:
This line is changed from being on `icon-button-icon` to its parent here; is that intentional?

Reply:
@timabbott No? Both `icon-button.primary` and `.icon-button.primary .icon-button-icon` are different. I guess that's what you meant?

Reply:
Right, exactly, which I think means this commit isn't a noop refactor; can you fix it to be?

Reply:
All I did was reorder the `.icon-button-icon` to bring it up for proper nesting in this commit. I don't see an issue here or I ain't getting what you mean here, sorry :sweat: .

Reply:
Ahh. so I think the confusion here is that the diff looks a little confusing... specifically:
```diff
-#settings_page .icon-button.primary {
-    background-color: hsl(156, 30%, 50%);
+#settings_page .icon-button.primary .icon-button-icon {
+    font-weight: lighter;
     color: hsl(0, 0%, 100%);
-    border: 1px solid hsl(155, 30%, 50%);
+}
+
+#settings_page .icon-button.save-button.saving {
+    background-color: hsl(156, 14%, 40%);
+    border-color: hsl(156, 14%, 40%);
 }
 
 #settings_page .icon-button .icon-button-icon {
@@ -500,11 +510,6 @@ input[type=checkbox] {
     font-weight: 500;
 }
 
-#settings_page .icon-button.primary .icon-button-icon {
-    font-weight: lighter;
-    color: hsl(0, 0%, 100%);
-}
-
```

To elaborate:
```diff
-#settings_page .icon-button.primary {
-    background-color: hsl(156, 30%, 50%);
+#settings_page .icon-button.primary .icon-button-icon {
+    font-weight: lighter;
     color: hsl(0, 0%, 100%);
-    border: 1px solid hsl(155, 30%, 50%);
+}
+
```
seems to suggest you've erroneously added the `color: hsl(0, 0%, 100%);` to `.icon-button-icon`.... but actually we also have:
```diff
-#settings_page .icon-button.primary .icon-button-icon {
-    font-weight: lighter;
-    color: hsl(0, 0%, 100%);
-}
-
```
Which means it already had `color: hsl(0, 0%, 100%)` and you just moved it and it landed on a line with `color: hsl(0, 0%, 100%)` and this made the commit diff a little confusing. I think you can just add this explanation to your commit message and you should be good.

Atleast... that's as far as I understand?